README.md

```diff
@@ -110,12 +110,11 @@ Standard Apache common log output.
 
 ##### dev
 
-Concise output colored by response status for development use. The `:status`
-token will be colored red for server error codes, yellow for client error
-codes, cyan for redirection codes, and uncolored for all other codes.
+Concise output colored by response status for development use.
+See `:status-colored` for details.
```

Comment:
I'm not sure this part is needed, as why jot say to see all the different tokens? We should probably either keep the color explanation here or just remove this part, I think.

Reply:
agreed. I'll add the color explanation back in